dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/metadata/inputRenderer/SuggestionInputFieldRenderer.java

```diff
@@ -146,4 +149,19 @@ private Map<String, String> findFilterValues(DatasetField datasetField, Map<Stri
         return filteredValues.containsValue(StringUtils.EMPTY) ? new HashMap<>() : filteredValues;
     }
 
+    private Map<String, String> getFilterValue(DatasetField datasetField) {
+        return datasetFieldTypeToSuggestionFilterMapping
+                .entrySet()
+                .stream()
+                .filter(entry -> entry.getKey().equalsIgnoreCase(datasetField.getTypeName()))
+                .findFirst()
+                .map(entry -> {
+                    Map<String, String> filterValues = new HashMap<>();
+                    // Assuming getFieldValue() returns an Optional or similar
```

Comment:
Is this a leftover after refactoring or something?

Reply:
Removed